src/app/recipes/recipe.service.ts

```diff
@@ -0,0 +1,49 @@
+import { EventEmitter, Injectable } from "@angular/core";
+import { Ingredient } from "../shared/ingredient.model";
+import { ShoppingListService } from "../shopping-list/shopping-list.service";
+import { Recipe } from "./recipe.model";
+
+@Injectable()
+export class RecipeService {
+    recipeSelected = new EventEmitter<Recipe>();
+
+    private recipes: Recipe[] = [
+        new Recipe(
```

Comment:
Take the mock data from the mock file. You can have a mock data file under the resources or asset from where you can pick this values